scripts/build/entries/index.ts

```diff
@@ -0,0 +1,257 @@
+import { exec } from 'node:child_process';
+import { join } from 'node:path';
+
+import type { BuildEntriesByPackageName } from '../utils/entry-utils';
+
+export const buildEntries = {
+  storybook: {
+    prebuild: async (cwd) => {
+      const CORE_PREBUILD_SCRIPT_PATH = join(
+        import.meta.dirname,
+        'storybook',
+        'generate-source-files.ts'
+      );
+      return new Promise((resolve, reject) => {
+        const child = exec(`jiti ${CORE_PREBUILD_SCRIPT_PATH}`, {
```

Comment:
Aren't we using `execa` consistently anymore, but we are going back to native `exec` again? We should keep in mind that this may make development on Windows machines more challenging again. As long as we don't have a strong reason to ditch execa, we should ensure cross-platform support and keep using it.